NDB.Covid19/NDB.Covid19.iOS/Views/Welcome/WelcomePageWhatIsNew.storyboard

```diff
@@ -97,6 +97,12 @@ line 2</string>
                                                                     </label>
                                                                 </subviews>
                                                             </stackView>
+                                                            <view contentMode="scaleToFill" translatesAutoresizingMaskIntoConstraints="NO" id="kYi-se-jGo" userLabel="Spacer">
+                                                                <rect key="frame" x="0.0" y="357" width="374" height="50"/>
+                                                                <constraints>
+                                                                    <constraint firstAttribute="height" constant="50" id="2hw-1X-lpZ"/>
```

Comment:
Don't we have any standardized margins for this?

Reply:
Not on iOS that I'm aware of

Reply:
only on Android